Description: Set wx-config version
 Useful when there are multiple versions of wx-config installed on the system
Author: Fab Stz <fabstz-it@yahoo.fr>
Origin: Debian
Forwarded: https://freefilesync.org/forum/viewtopic.php?t=9685
Last-Update: 2022-08-12

--- a/FreeFileSync/Source/Makefile
+++ b/FreeFileSync/Source/Makefile
@@ -2,12 +2,13 @@
 exeName = FreeFileSync_$(shell arch)
 
 PKG_CONFIG=pkgconf
+WX_CONFIG=wx-config --version=3.2
 
 CXXFLAGS += -std=c++23 -pipe -DWXINTL_NO_GETTEXT_MACRO -I../.. -I../../zenXml -include "zen/i18n.h" -include "zen/warn_static.h" \
            -Wall -Wfatal-errors -Wmissing-include-dirs -Wswitch-enum -Wcast-align -Wnon-virtual-dtor -Wno-unused-function -Wshadow -Wno-maybe-uninitialized \
-           -O3 -DNDEBUG `wx-config --cxxflags --debug=no` -pthread
+           -O3 -DNDEBUG `$(WX_CONFIG) --cxxflags --debug=no` -pthread
 
-LDFLAGS += -s `wx-config --libs std, aui, richtext --debug=no` -pthread
+LDFLAGS += -s `$(WX_CONFIG) --libs std, aui, richtext --debug=no` -pthread
 
 
 CXXFLAGS  += `$(PKG_CONFIG) --cflags openssl`
--- a/FreeFileSync/Source/RealTimeSync/Makefile
+++ b/FreeFileSync/Source/RealTimeSync/Makefile
@@ -2,12 +2,13 @@
 exeName = RealTimeSync_$(shell arch)
 
 PKG_CONFIG=pkgconf
+WX_CONFIG=wx-config --version=3.2
 
 CXXFLAGS += -std=c++23 -pipe -DWXINTL_NO_GETTEXT_MACRO -I../../.. -I../../../zenXml -include "zen/i18n.h" -include "zen/warn_static.h" \
            -Wall -Wfatal-errors -Wmissing-include-dirs -Wswitch-enum -Wcast-align -Wnon-virtual-dtor -Wno-unused-function -Wshadow -Wno-maybe-uninitialized \
-           -O3 -DNDEBUG `wx-config --cxxflags --debug=no` -pthread
+           -O3 -DNDEBUG `$(WX_CONFIG) --cxxflags --debug=no` -pthread
 
-LDFLAGS += -s `wx-config --libs std, aui, richtext --debug=no` -pthread
+LDFLAGS += -s `$(WX_CONFIG) --libs std, aui, richtext --debug=no` -pthread
 
 #Gtk - support "no button border"
 CXXFLAGS  += `$(PKG_CONFIG) --cflags gtk+-2.0`
